tools/Importer/readme.md

```diff
@@ -0,0 +1,32 @@
+# Importer
+
+The Importer is a .net console application that loads data to the FHIR server at high speed. It improves the data loading speed significantly through batch multi-thread processing. The tool is very easy to use. Download the code, update the settings in the app.config file, compile and run.
+
+Note that you can use the built-in bulk import ($import) feature in the managed service of Azure Health Data Services. For more info on other data loading tools and options, check the [documentation](https://docs.microsoft.com/azure/healthcare-apis/fhir/bulk-importing-fhir-data).
+
+## Features
+
+The primary goal of the Importer is to help load data to the FHIR server as quickly as it can. While the app.config file contains detailed information on the settings, it's worth mentioning some key features of the tool.
+
+- Process ndjson files and resources (one resource per line) in each ndjson file in batch parallel operation.
+- Support one or multiple endpoints pointing to the same database. 
+- Read blobs/files stored in Azure storage container from specific start and end positions. For example, if there are one million files, you can skip 100,000 files, and load the next 50,000 files.
+- Define parallel read threads (ReadThreads) and write threads (WriteThreads). Within each read thread, the write threads is the ratio of (WriteThreads / ReadThreads).
+- Define batches for read threads (e.g. 19 files in a batch) and write threads (e.g. 100 lines in a batch from an ndjson file).
+- Report reads and writes data for a defined period time, e.g. 30 seconds.
+- Retry if an error is encountered, e.g. 10 times.
+

```

Comment:
We need to add expectations as fat as throughput. Tool loads resources one-by-one. This is not limiting factor for cosmos backend but is for SQL one.

Reply:
To clarify, the Importer's performance is subject to the limit due to "one by one" resource loading when SQL db is used. This limit does not impact performance as much when Cosmos DB is used. Right?

Reply:
I will revise the doc per our chat.